job_tracker/app/models/job_application.rb

```diff
@@ -1,10 +1,9 @@
 class JobApplication < ApplicationRecord
   belongs_to :employer
-  has_one :job_description, dependent: :delete, required: true
```

Comment:
This is now handled by the form objects & controllers. JobApplication won't be saved without a valid JobDescription. Removing this allows rpsec and rails console to create test objects